pybossa/repositories/task_repository.py

```diff
@@ -315,10 +315,9 @@ def update_task_state(self, project_id, n_answers):
                    AND task.project_id=:project_id
                    AND task.calibration!=1
                    GROUP BY task.id
-                   having COUNT(task_run.id) >=:n_answers);
+                   having COUNT(task_run.id) >= task.n_answers);
```

Comment:
n_answers param passed to update_task_state is unused and can be removed. looks good otherwise.